### DISCUSSION TITLE
了解ai運作

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1604] [General]
**此篇目的為了解此網站ai是如何判斷的**

ai就是人工智慧，要經過訓練和學習才能高準確的判斷正確，但或許會有一些漏網之魚，身為一個好奇的人，我很想了解這裡有哪些不足的地方。

我大概能想到幾個判斷的標準，像是判斷贅詞、是否抄襲、有沒有原創性及個人想法或是字數和圖片多寡等等，大家如果有什麼其他想法可以在底下留言或測試。

下圖為ai的示意圖
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105231392/8c6e68e7-52ca-4f24-b60a-8cd16805eba9)

(如果期待落空了請見諒)

**Comment by lazywulf** [comment]
是否能請版主解釋您打算如何以其回應知道模型的判斷方式？
我覺得這是很有趣的，本人只知道鮮少關於對抗式攻擊等等的東西，非常希望版主可以回應並介紹。
祝版主成功。
